README.md

```diff
@@ -15,6 +16,12 @@ The kartograafr application...
 * can be easily updated by changing its configuration course page in Canvas,
 * allows support staff to update its configuration via Canvas without requiring server sysadmin access
 
+## Conventions:
+* The git workflow is to use feature branches for development.
+* At UMich kartograafr runs in Docker and for production we run Docker using OpenShift. See below for more information.
+* Configuration changes are done by checking new configuration into github, creating a new tag and build and redeploying in OpenShift.
+_Sensitive_ information is keep in OpenShift secrets and is not part of the build.
```

Comment:
Are the secrets stored in a (private) repo?  If not, then where?

I'm looking at mostly just the highlighted changes in this PR.  I looked at the full document, too, and the information about secrets is vague.  I'd like to see more details.